content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,236 @@
+
+## Configurando Knobs
+
+Knobs es un recurso increíble para que los diseñadores y desarrolladores experimenten y jueguen con componentes en un entorno controlado sin necesidad de codificar! Básicamente, proporciona campos definidos dinámicamente con los que un usuario manipula los props que se pasan a los componentes de sus historias. Esto es lo que vamos a implementar ...
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Instalación
+
+Primero, tendremos que agregarlo como una dependencia de desarrollo
+
+```bash
+yarn add -D @storybook/addon-knobs
+```
+
+Registra Knobs en tu archivo `.storybook/main.js`.
+
+```javascript
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: ['@storybook/addon-actions', '@storybook/addon-links', '@storybook/addon-knobs'],
+};
+```
+
+<div class = "aside">
+<strong> 📝 ¡El orden de registro de complementos es importante! </strong>
+<br/>
+El orden en que enumere estos complementos determinará el orden en que aparecerán como pestañas en su panel de complementos (para aquellos que aparecen allí).
+</div>
+
+¡Eso es! Es hora de usarlo en una historia.
+
+### Uso
+
+Usemos el tipo de knob de objeto en el componente `Task`.
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de la exportación `default` del archivo`Task.stories`, agregue `withKnobs` como elemento de `decorators`:
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  title: 'Task',
+  decorators: [withKnobs],
+  // same as before
+};
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: object('task', { ...taskData }),
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+// archived task state
+export const Archived = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_ARCHIVED',
+      },
+    },
+  },
+  methods: actionsData,
+});
```

Comment:
@IvanSotelo in this snippet, the `pinned` and `archived` need to match the simple component section, to avoid introducing back issues with jest/vue cli